README.md

```
@@ -1,16 +1,20 @@
-# edu_bridge_app
+# EduBridge: Democratizing Education in Bangladesh 🎓
 
-A new Flutter project.
+<div align="center">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: div

(MD033, no-inline-html)